influxdb/client.py

```diff
@@ -84,6 +82,8 @@ def __init__(self,
         self.udp_port = udp_port
         if use_udp:
             self.udp_socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        else:
```

Comment:
lol I like how the tests use this when publishing a UDP metric does not. ;)

I'll fix this up.